NuGet.Config

```diff
@@ -4,7 +4,8 @@
     <add key="All" value="(Aggregate source)" />
   </activePackageSource>
   <packageSources>
-    <add key="Public Feed" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+    <!--<add key="Public Feed" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />-->
+    <add key="Public Feed" value="https://api.nuget.org/v2/ />
```

Comment:
Oops.  I was getting an error restoring gitlink when I tried to build, and this change seemed to fix it.  I'll rebase it out.